### PR TITLE
New Armenian Cheat Sheet

### PR DESCRIPTION
**Pull request by lerna**
**What does your Instant Answer do?**
It's a goodie providing translations of basic armenian words and phrases including phonetic pronunciations.

**What problem does your Instant Answer solve (Why is it better than organic links)?**
There is no current Armenian cheat sheet in DDG.  This will allow users a quick and easy way to get started with some basic armenian words.

**What is the data source for your Instant Answer? (Provide a link if possible)**
http://www.omniglot.com/language/phrases/westernarmenian.htm

**Why did you choose this data source?**
It provides a quick overview of most common Armenian basic words and phrases.

**Are there any other alternative (better) data sources?**
There are but they are more complicated for a non Armenian speaker to decipher.

**What are some example queries that trigger this Instant Answer?**
"armenian cheat sheet"

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
Tourists, travelers, people interested in learning basic armenian

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**
No

**Which existing Instant Answers will this one supercede/overlap with?**
None

**Are you having any problems? Do you need our help with anything?**
No

**Where did you hear about DuckDuckHack? (For first time contributors)**
Coder Day of Service organized by {code} montage 

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**
screen shot 2015-11-08 at 11 32 17 am

## Checklist
Please place an 'X' where appropriate.

```
- [X] Added metadata and attribution information (https://duck.co/duckduckhack/metadata)
- [] Wrote test file and added to t/ directory (https://duck.co/duckduckhack/goodie_tests)
- [X] Verified that instant answer adheres to design guidelines (https://duck.co/duckduckhack/design_styleguide)
- [X] Verified that instant answer adheres to code styleguide (https://duck.co/duckduckhack/code_styleguide)
- [X] Tested cross-browser compatibility
    Please let us know which browsers/devices you've tested on:
    - Windows 10
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] IE 11
        - [] Microsoft edge
        
    - Windows 8
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] IE 10

    - Windows 7
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] IE 8
        - [] IE 9
        - [] IE 10

    - Windows XP
        - [] IE 7
        - [] IE 8

    - Linux
        - [] Google Chrome
        - [] Firefox
        
    - Mac OSX
        - [X] Google Chrome
        - [] Firefox
        - [] Opera
        - [X] Safari

    - iOS (iPhone)
        - [] Safari Mobile
        - [] Google Chrome
        - [] Opera

    - iOS (iPad)
        - [] Safari Mobile
        - [] Google Chrome
        - [] Opera

    - Android
        - [] Firefox
        - [] Native Browser
        - [] Google Chrome
        - [] Opera

```

-----
IA Page: https://duck.co/ia/view/armenian_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@lerna Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

File: share/goodie/cheat_sheets/json/armenian.json [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+    "metadata" : {
+        "sourceName" : "omniglot",
+        "sourceUrl" : "http://www.omniglot.com/language/phrases/westernarmenian.htm"
+    },
+
+    "aliases": [
+        "armenian phrases", "english to armenian", "basic armenian phrases", "basic armenian"
+    ],
+
+    "template_type": "reference",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/armenian.json]: We actually *just* made a new template for these language Cheat Sheets that supports a third property for the pronunciation/transliteration.

Can you please switch this IA to use that template? Here's an example of an IA using that template: https://github.com/iambibhas/zeroclickinfo-goodies/commit/cca073c3d1ae9a0bbd33501eae50e146d206eaf0

Comment (lerna) [inline share/goodie/cheat_sheets/json/armenian.json]: Sounds good.  I will update it.

**Comment by lerna** [comment]
Updated the new armenian cheat sheet to use the language template and moved it under the new language cheat sheets directory.

**Comment by tagawa** [comment]
Feedback comment from Twitter:
I would make "Եւ դուք ինչպէ՞ս էք" into "Իսկ դուք ինչպե՞ս էք" or "Դուք ինչպե՞ս էք"

**Comment by vgratian** [comment]
That was me :) "Եւ" is correct but redundant.

**Comment by daxtheduck** [comment]
Automated screenshot from https://beta.duckduckgo.com
                
                mobile: https://ia-screenshots.s3.amazonaws.com/armenian_cheat_sheet_mobile.png

**Comment by lerna** [comment]
Agreed @vgratian.  Updated it now and got rid of "Եւ".  Also updated it to a more common way of saying it in armenian.

**Comment by moollaza** [comment]
@lerna this LGTM :+1:

Thanks a lot for contributing (we really appreciate it :smiley:). We invite you to stick around and comment/help on other PR's & Issues or even submit an Instant Answer!

File: share/goodie/cheat_sheets/json/language/armenian.json [hunk 2]
```diff
@@ -0,0 +1,219 @@
+{
+    "id" : "armenian_cheat_sheet",
+
+    "name" : "Armenian Cheat Sheet (Western)",
+
+    "metadata" : {
+        "sourceName" : "omniglot",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/language/armenian.json]: Nit: Capitalize `Omniglot`

**Comment by moollaza** [comment]
@lerna congrats! It's live: https://duckduckgo.com/?q=Armenian+Cheat+Sheet&ia=cheatsheet

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's, tackle open issues or even submit more Instant Answers!

**Pull request merged**